engine/api/src/main/java/org/opennms/oce/engine/api/Engine.java

```diff
@@ -50,8 +50,16 @@
     /**
      * Passes the reference to the SituationHandler.
      * The SituationHandler exposes <code>onSituation()</code> callback for creating and updating Situations.
+     *
      * @param handler
      */
     void registerSituationHandler(SituationHandler handler);
 
+    /**
+     * Delete the situation with the given situation Id.
+     *
+     * @param situationId the situation Id
+     */
+    default void deleteSituation(String situationId) {
```

Comment:
I'd avoid adding a default here and let the implementation declare it explicitly.